client/js/components/tags/TagsList.vue

```diff
@@ -235,8 +225,9 @@ export default {
     deleteItem (item) {
       dpRpc('bulk.delete.tags.and.topics', { ids: [item] })
         .then(checkResponse)
-        .then(response => {
-          console.log(response, 'response delete')
+        .then(() => {
+          this.dataIsRequested = false
```

Comment:
i would remove  this here again, but add it in line 257 in a new `then()`. Otherwise we undermine the mechanism to prevent the race condition.